.gitignore

```diff
@@ -1,3 +1,5 @@
+local.settings.json
```

Comment:
Should this be under `# User-specific files` (line 45)?

Reply:
https://github.com/Particular/PlatformArchitecture/pull/506#discussion_r438676055 🙂 

Sure, let's keep the new line.